packages/test-generator/integration-test-templates/cypress/e2e/update-form-spec.cy.ts

```diff
@@ -44,6 +44,14 @@ describe('UpdateForms', () => {
         });
         clickAddToArray();
 
+        // Belongs to update
+        removeArrayItem('John');
+        getArrayFieldButtonByLabel('Belongs to owner').click();
+        cy.get(`.amplify-autocomplete`).within(() => {
+          cy.get('input').type(`John{downArrow}{enter}`);
```

Comment:
Maybe change to something not John so we know update's working?

Reply:
Right, good point